shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/builder/SchemaBuilder.java

```diff
@@ -147,9 +173,35 @@ private static void appendDialectRemainTables(final DialectTableMetaDataLoader d
             }
         }
     }
-    
+
+    private static Map<String, String> getActualTable2LogicTableMap(final SchemaBuilderMaterials materials, final Collection<String> ruleLogicTables) {
+        List<DataNodeContainedRule> dataNodeContainedRuleList = materials.getRules().stream()
+                .filter(each -> each instanceof DataNodeContainedRule)
+                .map(each -> (DataNodeContainedRule) each)
+                .collect(Collectors.toList());
+        if (CollectionUtils.isEmpty(dataNodeContainedRuleList)) {
+            return Collections.emptyMap();
+        }
+
+        int size = dataNodeContainedRuleList.stream()
+                .mapToInt(each -> each.getAllDataNodes().entrySet().stream()
+                        .mapToInt(entry -> entry.getValue().size())
+                        .sum())
+                .sum();
+        Map<String, String> actualTable2LogicTableMap = new HashMap<>(size);
```

Comment:
Please refer official code conduct to change the code style.

Reply:
Do you mean that change 'entry' to 'each'? But it is in a map.